core-aam/index.html

```diff
@@ -9852,8 +9852,15 @@ <h3>Group Position</h3>
             <a class="property-reference" href="#aria-level"><code>aria-level</code></a
             >, <a class="property-reference" href="#aria-posinset"><code>aria-posinset</code></a
             >, and <a class="property-reference" href="#aria-setsize"><code>aria-setsize</code></a> are all 1-based. When the [=ARIA/property=] is not present or is &quot;0&quot;, it indicates the
-            property is not computed or not supported. If any of these properties are specified by the author as either &quot;0&quot; or a negative number, [=user agents=] SHOULD use &quot;1&quot;
-            instead.
+            property is not computed or not supported.
+          </p>
+          <p>
+            If the <a class="property-reference" href="#aria-level"><code>aria-level</code></a> or <a class="property-reference" href="#aria-posinset"><code>aria-posinset</code></a> properties are
+            specified by the author as either &quot;0&quot; or a negative number, [=user agents=] SHOULD use &quot;1&quot; instead.
+          </p>
+          <p>
+            If the <a class="property-reference" href="#aria-setsize"><code>aria-setsize</code></a> property is specified by the author as any string value other than a positive whole number, the set
```

Comment:
actually, "" is not indeterminate either.

Reply:
yeah, that's covered in another part of the spec.  guess this should have said "non-empty string value..."  i'll change it to that.